src/hotspot/share/c1/c1_LinearScan.hpp

```diff
@@ -31,6 +31,7 @@
 #include "c1/c1_Instruction.hpp"
 #include "c1/c1_LIR.hpp"
 #include "c1/c1_LIRGenerator.hpp"
+#include "compiler/oopMap.hpp"
```

Comment:
The description states:
> There are only 4 header files that include oopMap.hpp. All of these can be replaced for forward declarations.

But this had to be added. Did you check how many files include c1_LinearScan.hpp?

Reply:
> The description states:
> 
> > There are only 4 header files that include oopMap.hpp. All of these can be replaced for forward declarations.
> 
> But this had to be added. Did you check how many files include c1_LinearScan.hpp?

c1_LinearScan.hpp is included in only 6 .o files.

Reply:
Great!